src/context/authContext.tsx

```
@@ -0,0 +1,41 @@
+  children: React.ReactNode
+}
+
+export const AuthContext = createContext({} as AuthContextType);
+
+export function AuthProvider({ children }: AuthProviderProps) {
+  const {
+    authenticated,
+    authenticatedUser,
+    handleLogin,
+    handleLogout,
+  } = useAuth()
```

ESLint

[error] Replace `⏎····authenticated,⏎····authenticatedUser,⏎····handleLogin,⏎····handleLogout,⏎··}·=` with `·authenticated,·authenticatedUser,·handleLogin,·handleLogout·}·=⏎···`

(prettier/prettier)